index.src.html

```diff
@@ -233,8 +231,29 @@ <h2 id="terms">Key Concepts and Terminology</h2>
 <section>
   <h2 id="referrer-policies" oldids="referrer-policy-states">Referrer Policies</h2>
 
-  Each possible <a>referrer policy</a>, besides the empty string, is explained
-  below. A detailed algorithm for evaluating their effect is given in the
+  A <dfn title="concept-referrer-policy" data-export>referrer policy</dfn>
+  is the empty string, "<code>no-referrer</code>",
+  "<code>no-referrer-when-downgrade</code>", "<code>same-origin</code>",
+  "<code>origin</code>", "<code>strict-origin</code>",
+  "<code>origin-when-cross-origin</code>",
+  "<code>strict-origin-when-cross-origin</code>", or
+  "<code>unsafe-url</code>".
+
+<pre class=idl>enum <dfn data-dfn-type=enum data-export>ReferrerPolicy</dfn> {
+
```

Comment:
Oh wow, I have no idea why that would happen either. Oh well! Sorry for not checking the output.